docs/spherical-video-v2-rfc.md

```diff
@@ -267,3 +267,155 @@ monoscopic equirectangular video:
               ...
 ```
 
+### WebM (Matroska)
+Spherical video metadata is stored in a new master element, `Projection`,
+placed inside a video track's `Video` master element.
+
+
+As the V2 specification stores its metadata in a different location, it is
+possible for a file to contain both the V1 and V2 metadata. If both V1 and
+V2 metadata are contained they should contain semantically equivalent
+information, with V2 taking priority when they differ.
+
+#### `Projection` master element
+##### Definition
+ID: 0x7670 \
+Level: 4 \
+Mandatory: No \
+Type: Master  \
+Default: N/A \
+Minver: 4 \
+WebM: Yes \
+Container: `Video` master element
+
+Describes the video projection details. Used to render spherical and VR videos.
+
+#### `ProjectionType` element
+##### Definition
+ID: 0x7671 \
+Level: 5 \
+Mandatory: Yes \
+Type: uinteger \
+Default: 0 \
+Minver: 4 \
+WebM: Yes \
+Container: `Projection` master element
+
+Describes the projection used for this video track.
+
+##### Semantics
+`ProjectionType` is an enum. The valid values are:
+
+* 0: Rectangular
+* 1: Equirectangular
+* 2: Cubemap
+
+
+#### `ProjectionPrivate` element
+##### Definition
+ID: 0x7672 \
+Level: 5 \
+Mandatory: No \
+Type: binary  \
+Default: N/A \
+Minver: 4 \
+WebM: Yes \
+Container: `Projection` master element
+
+Private data known only to the projection codec.
+
+##### Semantics
+ * If `ProjectionType` equals 0 (Rectangular), then this element must not be
+present.
+ * If `ProjectionType` equals 1 (Equirectangular), then this element must be
+ present and contain the same binary data that would be stored inside an
+ISOBMFF Equirectangular Projection Box ('equi').
+ * If `ProjectionType` equals 2 (Cubemap), then this element must be present
+and contain the same binary data that would be stored inside an ISOBMFF
+Cubemap Projection Box('cbmp').
+
+Note: ISOBMFF box size and fourcc fields are not included in the binary
+data, but the FullBox version and flag fields are. This is to avoid
+redundant framing information while preserving versioning and semantics
+between the two container formats.
+
+#### `ProjectionPoseYaw` element
```

Comment:
I don't see any other examples in Matroska that puts units in the name. I don't know if there is actually examples of it in ISOBMFF either, but I'm not focusing on that right now. DisplayWidth and DisplayHeight can technically have alternate units specified via DisplayUnits. I suspect if we ever needed to allow multiple unit types for pose, we'd follow a similar pattern.